### PR TITLE
Move bootstrap and project hardcoded values  to variables

### PR DESCRIPTION
**Pull request by jaroslawgajewski**
`Boostrap` and `Org` level labels moved to variables
Corresponding Readme files updated

### CODE REVIEW

**Review by bharathkkb — COMMENTED** [review]
Thanks for the PR @jaroslawgajewski 

One of the reasons why we have not parameterized this is because we expect users to fork and make code changes to adapt this repo for their organization. We also did not want to over parameterize too much. Was there any blockers in forking and changing the labels directly?

**Comment by jaroslawgajewski** [comment]
@bharathkkb 
Main reason is to provide an option to keep code inline with example foundation repo and adjust it for own purpose based on variables. That should allow us to make code more flexible and inline with good practices.

**Comment by daniel-cit** [comment]
Not all labels can be freely changed without consequences. 
Labels `application_name` and `environment` are used in the code to do a lookup for projects created in previous steps.
To make the labels really configurable they need to be consistent in all steps and they need to address theses and other lookups:

https://github.com/terraform-google-modules/terraform-example-foundation/blob/1fad10b2af93069a7369cd100a3fcf2cddaa6f74/4-projects/modules/single_project/data.tf#L19

https://github.com/terraform-google-modules/terraform-example-foundation/blob/1fad10b2af93069a7369cd100a3fcf2cddaa6f74/3-networks/envs/non-production/main.tf#L83

https://github.com/terraform-google-modules/terraform-example-foundation/blob/1fad10b2af93069a7369cd100a3fcf2cddaa6f74/3-networks/envs/non-production/main.tf#L91

https://github.com/terraform-google-modules/terraform-example-foundation/blob/1fad10b2af93069a7369cd100a3fcf2cddaa6f74/3-networks/modules/vpn-ha/main.tf#L34

https://github.com/terraform-google-modules/terraform-example-foundation/blob/1fad10b2af93069a7369cd100a3fcf2cddaa6f74/4-projects/business_unit_1/production/example_peering_project.tf#L22

https://github.com/terraform-google-modules/terraform-example-foundation/blob/1fad10b2af93069a7369cd100a3fcf2cddaa6f74/3-networks/modules/restricted_shared_vpc/dns.tf#L31

https://github.com/terraform-google-modules/terraform-example-foundation/blob/1fad10b2af93069a7369cd100a3fcf2cddaa6f74/3-networks/modules/base_shared_vpc/main.tf#L30

https://github.com/terraform-google-modules/terraform-example-foundation/blob/1fad10b2af93069a7369cd100a3fcf2cddaa6f74/3-networks/envs/shared/net-hubs.tf#L43

https://github.com/terraform-google-modules/terraform-example-foundation/blob/1fad10b2af93069a7369cd100a3fcf2cddaa6f74/3-networks/envs/shared/net-hubs.tf#L52

https://github.com/terraform-google-modules/terraform-example-foundation/blob/1fad10b2af93069a7369cd100a3fcf2cddaa6f74/5-app-infra/modules/env_base/data.tf#L18

https://github.com/terraform-google-modules/terraform-example-foundation/blob/1fad10b2af93069a7369cd100a3fcf2cddaa6f74/5-app-infra/modules/env_base/data.tf#L26

File: 0-bootstrap/main.tf [hunk 1]
```diff
@@ -140,21 +133,6 @@ module "cloudbuild_bootstrap" {
...
-    environment       = "bootstrap"
-    application_name  = "cloudbuild-bootstrap"
-    billing_code      = "1234"
-    primary_contact   = "example1"
-    secondary_contact = "example2"
-    business_code     = "abcd"
-    env_code          = "b"
-  }
-
-  terraform_apply_branches = [
```
Comment (daniel-cit) [inline 0-bootstrap/main.tf:153]: The branch names cannot be changes without changing the wrapper script

https://github.com/terraform-google-modules/terraform-example-foundation/blob/1d5b05bf35e98d62b4145a349d490df872449dee/build/tf-wrapper.sh#L26

Comment (jaroslawgajewski) [reply 0-bootstrap/main.tf:153]: @daniel-cit 
should I add all above constraints into corresponding README files? 
As soon as we will move that data to variables forking and future code updates become easier I suppose.

**Pull request closed without merging**